stdlib/public/Concurrency/TaskCancellation.swift

```diff
@@ -34,8 +34,8 @@ public func withTaskCancellationHandler<T>(
 ) async rethrows -> T {
   let task = Builtin.getCurrentAsyncTask()
 
-  // unconditionally add the cancellation record to the task.
-  // if the task was already cancelled, it will be executed right away.
+  // Unconditionally add the cancellation record to the task.
+  // If the task was already canceled, it executes immediately.
```

Comment:
"If the task was already canceled, it executes immediately."

This makes it sound like "the task executes immediately", which I presume is not the case for a task that was already canceled.

Reply:
Nice catch - this is a `//` comment, not part of the documentation, so should not be changed by these edits.  I did a visual check for these for commit 6a79590ca67fef5c9d05debd253588437e96e99d; and I'll run my usual `grep` script to check for others now.

Reply:
Reverted in a0e58c68860.  @ktoso or @DougGregor do you want to work with @tbkka to improve this internal comment as needed?